microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -45,136 +46,140 @@ This article uses five demo webpages, all sourced at https://github.com/Microsof
+Heap snapshots can be viewed in multiple different ways in the **Memory** tool. Each way corresponds to a different task:
 
-**Summary view** shows objects grouped by the constructor name.  Use it to hunt down objects (and the memory use) based on type grouped by constructor name.  **Summary view** is particularly helpful for **tracking down DOM leaks**.
+* The **Summary** view shows objects grouped by their constructor name.  Use it to find objects, and the memory they use, based on types grouped by constructor name. The **Summary** view is particularly helpful for **tracking down DOM leaks**.
 
 <!--todo: add profile memory problems memory diagnosis (tracking down DOM leaks) section when available  -->
 
-**Comparison view**.  Displays the difference between two snapshots.  Use it to compare two (or more) memory snapshots from before and after an operation.  Inspecting the delta in freed memory and reference count lets you confirm the presence and cause of a memory leak.
+* The **Comparison** view displays the differences between two snapshots. Use it to compare two (or more) memory snapshots from before and after an operation.  Inspecting the delta in freed memory and reference count lets you confirm the presence and cause of a memory leak.
 
-**Containment view**.  Allows exploration of heap contents.  **Containment view** provides a better view of object structure, helping analyze objects referenced in the global namespace (window) to find out what is keeping objects around.  Use it to analyze closures and dive into your objects at a low level.
+* The **Containment** view allows the exploration of the heap contents.  The **Containment** view provides a better view of object structure, helping analyze objects referenced in the global namespace (window) to find out what is keeping objects around.  Use it to analyze closures and dive into your objects at a low level.
 
-To switch between views, use the selector at the top of the view.
+To switch between views, use the dropdown list at the top of the **Memory** tool:
 
 ![Switch views selector](./heap-snapshots-images/heap-snapshots-view-dropdown.png)
 
 > [!NOTE]
 > Not all properties are stored on the JavaScript heap.  Properties implemented using getters that run native code aren't captured.  Also, non-string values such as numbers aren't captured.
 
-### Summary view
+<!-- ------------------------------ -->
+#### Summary view
 
-Initially, a snapshot opens in the Summary view, displaying object totals, which can be expanded to show instances:
+Initially, a heap snapshot opens in the **Summary** view, which displays a list of constructors. Each constructor in the list can be expanded to show the objects that were instantiated using that constructor:
 
 ![Summary view](./heap-snapshots-images/heap-snapshots-constructor-retainers.png)
 
-Top-level entries are "total" lines.
+Top-level entries in the list of constructors are _total lines_.
```

Comment:
needs rewrite.  check rendered section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots?branch=pr-en-us-3033#summary-view

are we talking about UI string "total lines"?  or terminology in the industry?